src/components/Card/Card.tsx

```diff
@@ -313,6 +325,14 @@ export const Card = chakra((props: React.PropsWithChildren<CardProps>) => {
         <Box className="card-body" __css={styles.body}>
           {cardContents}
         </Box>
+        {cardRightContents.length ? (
+          <Box
+            className="card-right"
+            __css={{ ...styles.body, marginLeft: "m" }}
```

Comment:
The styles for this container need to be cleaned up.

Mobile View
- `width` should be `100%`
- `marginLeft` should be  `0`
- `marginTop` should be  `xs` (8px)

Desktop View
- `width` should be `180px`
- `marginLeft` should be `m`
- `marginTop` should be  `0`